tests/integration/cli/test_rituals.py

```
@@ -132,6 +209,180 @@ def test_unknown_ritual_is_a_usage_error(tmp_path, monkeypatch, capsys):
+        _write(below / "rituals", _PACKAGE)
+        monkeypatch.chdir(below)
+
+        exit_code = rituals_list()
+
+        assert exit_code == _USAGE_EXIT
+        assert str(below / "rituals.py") in capsys.readouterr().err
+
+    # What per-module origins exist for: two files collide by path, two
+    # submodules of one package collide by dotted name.
+    @staticmethod
+    def test_two_submodules_claiming_one_step_name_collide(
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_two_submodules_claiming_one_step_name_collide`

Add return type annotation: `None`

(ANN205)